### PR TITLE
Add Code Styling to shared managed code

### PR DESCRIPTION
**Pull request by macrogreg**
This applies the same styles as https://github.com/DataDog/dd-continuous-profiler-dotnet/pull/126

### CODE REVIEW

File: samples/Datadog.AutoInstrumentation.ManagedLoader.Demo/Driver/Datadog.AutoInstrumentation.ManagedLoader.Demo.Driver/public/Program.cs [hunk 1]
```diff
@@ -133,12 +130,12 @@ private bool LoadTestTargetsInCurrentAppDomain()
                 ConsoleWrite.LineLine("Invoking AssemblyLoader.Run(..)");
                 ConsoleWrite.Line($"    Current AD: Id={AppDomain.CurrentDomain.Id}, IsDefaultAppDomain={AppDomain.CurrentDomain.IsDefaultAppDomain()}.");
 
-                var assemblyNamesToLoadIntoDefaultAppDomain = new string[] { "Datadog.AutoInstrumentation.ManagedLoader.Demo.Driver.DefAd.Asm1.dll" };
-                var assemblyNamesToLoadIntoNonDefaultAppDomains = new string[] { "Datadog.AutoInstrumentation.ManagedLoader.Demo.Driver.DefAd.Asm1.dll" };
+                string[] assemblyNamesToLoadIntoDefaultAppDomain = new string[] { "Datadog.AutoInstrumentation.ManagedLoader.Demo.Driver.DefAd.Asm1.dll" };
+                string[] assemblyNamesToLoadIntoNonDefaultAppDomains = new string[] { "Datadog.AutoInstrumentation.ManagedLoader.Demo.Driver.DefAd.Asm1.dll" };
```
Comment (gleocadie) [inline samples/Datadog.AutoInstrumentation.ManagedLoader.Demo/Driver/Datadog.AutoInstrumentation.ManagedLoader.Demo.Driver/public/Program.cs:134]: does it mean we will have to have the type everywhere even if it's obvious ?

File: samples/Datadog.AutoInstrumentation.ManagedLoader.Demo/Driver/Datadog.AutoInstrumentation.ManagedLoader.Demo.Driver/public/Program.cs [hunk 2]
```diff
@@ -176,13 +173,13 @@ private static bool IsDummyWorkPerformed(AppDomain appDomain, out string details
...
 
-            if (! (adData is string adDataString))
+            if (!(adData is string adDataString))
             {
                 details = $"Target AppDomain has Data under the expected key, however the type is {adData.GetType().FullName} whereas {typeof(string).FullName} was expected.";
                 return false;
             }
 
-            if (!adData.Equals(TestDataExpectedValue))
+            if (false == adData.Equals(TestDataExpectedValue))
```
Comment (gleocadie) [inline samples/Datadog.AutoInstrumentation.ManagedLoader.Demo/Driver/Datadog.AutoInstrumentation.ManagedLoader.Demo.Driver/public/Program.cs:182]: we still keep this ?

File: src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/internal/AssemblyResolveEventHandler.cs [hunk 3]
```diff
@@ -54,7 +57,7 @@ internal IReadOnlyList<string> ManagedProductBinariesDirectories
 
         private static AssemblyName ParseAssemblyName(string fullAssemblyName)
         {
-            if (string.IsNullOrEmpty(fullAssemblyName))
+            if (String.IsNullOrEmpty(fullAssemblyName))
```
Comment (gleocadie) [inline src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/internal/AssemblyResolveEventHandler.cs:60]: is it a suggestion in that case ? (`String` over `string`) I hope it's not an error.

File: src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/internal/AssemblyResolveEventHandler.cs [hunk 4]
```diff
@@ -86,12 +89,12 @@ private Assembly OnAssemblyResolveCore(AssemblyName assemblyName)
             }
 
             // Is this an assembly which should be loaded from the product directory?
-            if (! MustLoadAssemblyFromProductDirectory(assemblyName))
+            if (false == MustLoadAssemblyFromProductDirectory(assemblyName))
```
Comment (gleocadie) [inline src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/internal/AssemblyResolveEventHandler.cs:92]: same

File: src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/public/AssemblyLoader.cs [hunk 5]
```diff
@@ -326,7 +329,7 @@ private static IReadOnlyList<string> CleanAssemblyNamesToLoad(string[] assemblyN
                 return assemblyNamesToLoad;
             }
 
-            var validAssemblyNamesToLoad = new string[validAssemblyNamesCount];
+            string[] validAssemblyNamesToLoad = new string[validAssemblyNamesCount];
```
Comment (gleocadie) [inline src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/public/AssemblyLoader.cs:332]: same

File: src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/public/AssemblyLoader.cs [hunk 6]
```diff
@@ -340,7 +343,7 @@ private static IReadOnlyList<string> CleanAssemblyNamesToLoad(string[] assemblyN
 
         private static bool CleanAssemblyNameToLoad(string rawAssemblyName, out string cleanAssemblyName, out bool asmNameNeedsCleaning)
         {
-            if (string.IsNullOrWhiteSpace(rawAssemblyName))
+            if (String.IsNullOrWhiteSpace(rawAssemblyName))
```
Comment (gleocadie) [inline src/Shared-Libs/ManagedLoader/Datadog.AutoInstrumentation.ManagedLoader/public/AssemblyLoader.cs:346]: same

**Review by gleocadie — COMMENTED** [review]
I know you put a lot of effort in this but it does not reflect what we talked about.

**Comment by macrogreg** [comment]
> I know you put a lot of effort in this but it does not reflect what we talked about.

@gleocadie Yes, I figured out that what we wanted cannot work. We should sync up again. ))

**Pull request closed without merging**